packages/samples/mvp-dapp/src/actors/record.model.ts

```diff
@@ -42,8 +50,10 @@ export type StoreType = {
  */
 @ActorProvider({ ref: { name: 'record', path: '/:codeHash/:hashType/:args/' } })
 @LockPattern()
+@TypePattern()
 @DataPrefixPattern(DAPP_DATA_PREFIX)
```

Comment:
Is the `DataPrefixPattern` redundant now?

Reply:
It uses to filter cell's data that start with `DAPP_DATA_PREFIX`

Reply:
But `TypePattern` is added above, isn't it enough to filter cells of the `mvp contract`?

Reply:
You are right, it's enough.